development/contribute-to-docs.md

```diff
@@ -93,7 +93,7 @@ git commit -m “Add a better description of what a feature does”
 Then you have to upload your changes to GitHub:
 
 ```bash
-git push origin my-branch-name
+git push -f origin my-branch-name
```

Comment:
When committing normally, you shouldn't have to force push, only when changing existing commits